packages/app-staking/src/StakeList/Nominating.tsx

```diff
@@ -104,15 +108,32 @@ class Nominating extends React.PureComponent<Props> {
           onChange={this.onChangeNominee}
           value={nominee}
         />
+        { !isNomineeValid ?
+          t('nominator.error', {
+            defaultValue: 'The address you input is not intending to stake, and is therefore invalid. Please try again with a different address.'
+          })
+          :
+          null
+        }
+        { !isAddressFormatValid ?
+          t('nominator.error', {
+            defaultValue: 'The address you input does not conform to a recognized address format. Please make sure youve entered the address correctly and try again.'
+          })
+          :
+          null
+        }
       </Modal.Content>
     ];
   }
 
   private onChangeNominee = (nominee: string) => {
     const { intentions } = this.props;
 
+    const publicKey = addressDecode(nominee);
```

Comment:
Great. Small issue remains, the function throws, so just wrap in try/catch, see decoding at eg. https://github.com/polkadot-js/common/blob/master/packages/util-keyring/src/address/decode.ts#L16